source/about/cloud-subscriptions.rst

```diff
@@ -0,0 +1,114 @@
+
+Yes. Annual Cloud subscriptions need to be processed by our Sales team, as we don't currently support an annual term using the self-service Customer Portal or Cloud workspace.
+
+If you upgrade during a monthly billing cycle, charges will be prorated. For example, if you upgrade from monthly Starter to an annual term of Mattermost Professional on 6/15, we'll charge you on 7/1 for Starter for the period 6/1 - 6/14. A separate invoice will be provided for the annual Mattermost Professional subscription period starting 6/15.
+
+How do I cancel my subscription? 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To cancel your subscription, please `contact us <https://customers.mattermost.com/cloud/contact-us>`__.
+
+How much does Mattermost Cloud cost?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Mattermost Cloud Professional is $10 USD per user, per month as a pay-as-you-go subscription. If you have over 1000 users, please contact us for a quote.
+* Mattermost Cloud Enterprise is an annual subscription. Please contact us for a quote.
+
+How can customers trial Mattermost Cloud?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Mattermost Cloud Professional plan includes a 14-day free trial.
+* Mattermost Cloud Enterprise offers a discounted proof-of-concept program for a 60-day period. The proof-of-concept program includes a dedicated Customer Engineer.
+
+When will support for other regions be available?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Mattermost Cloud Enterprise will support data residency based on feedback from our customers. We appreciate feedback from our customers on regional support for both editions.
+
+How is Mattermost Cloud secured?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Mattermost Cloud has encryption at-rest and in-transit. We're SOC2 Type 1 certified and working towards SOC2 Type 2.
+
+Mattermost Cloud Enterprise is available to be deployed in a private environment within an AWS VPC dedicated to a single customer. This architecture provides the isolation and security of an on-premises deployment while ensuring consistent performance and uptime of your Mattermost workspace.
+
+What are the alternatives, and why should customers choose Mattermost Cloud?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Other SaaS offerings are closed-source and focus on a very broad use case. Mattermost Cloud is built on our open source core offering, allowing customers to view our source code and have a high degree of transparency into Mattermost development processes. Mattermost provides a DevOps-centric offering and has deep integrations and workflows that help builders and operators be more effective.
+
+Mattermost Cloud Enterprise is available to be deployed in a private environment within an AWS VPC dedicated to a single customer. Other SaaS offerings are multi-tenant, meaning customers share resources.
+
+Is Mattermost Cloud Enterprise a dedicated instance run on AWS systems?
```

Comment:
Some of this information about Mattermost Cloud Enterprise no longer apply. A dedicated AWS instance is a separate add-on to the regular Cloud Enterprise subscription (which is functionally equivalent to Self-Hosted Enterprise): https://docs.google.com/document/d/1nDO4NZwwxSNm_h-meJXitnNajETQ2rPam1X4D6bwfDo/edit

Reply:
Please review.  I added "can be" and "available as" and "contact sales for this option".  I think exposing this option is important to let them know the additional security benefit they can take advantage of.